workflows/chatbot/fine_tuning/finetune_neuralchat_v3/finetune_neuralchat_v3.py

```diff
@@ -0,0 +1,49 @@
+from optimum.habana import GaudiTrainingArguments
```

Comment:
move the code under workflows to https://github.com/intel/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/neural_chat/examples/finetuning